app/controllers/collection_templates_controller.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+class CollectionTemplatesController < CollectionDraftsController
+  include CollectionsCmrHelper
+  before_action :templates_enabled?
+  before_action :set_resource, only: %i[create_draft destroy edit update show]
```

Comment:
if these `before_action`s are already in the super class, you probably don't need to declare them again